README.md

```diff
@@ -111,47 +111,47 @@ We are building our npm distributions using Webpack to support legacy versions o
- [1.16.0](https://char0n.github.io/ramda-adjunct/1.16.0),
- [1.17.0](https://char0n.github.io/ramda-adjunct/1.17.0),
- [1.18.0](https://char0n.github.io/ramda-adjunct/1.18.0),
- [1.19.0](https://char0n.github.io/ramda-adjunct/1.19.0),
- [2.0.0](https://char0n.github.io/ramda-adjunct/2.0.0),
- [2.1.0](https://char0n.github.io/ramda-adjunct/2.1.0),
- [2.2.0](https://char0n.github.io/ramda-adjunct/2.2.0),
- [2.3.0](https://char0n.github.io/ramda-adjunct/2.3.0),
- [2.4.0](https://char0n.github.io/ramda-adjunct/2.4.0),
- [2.4.1](https://char0n.github.io/ramda-adjunct/2.4.1),
- [2.5.0](https://char0n.github.io/ramda-adjunct/2.5.0),
  [LATEST](https://char0n.github.io/ramda-adjunct)
+ [2.5.0](https://char0n.github.io/ramda-adjunct/2.5.0),
+ [2.4.1](https://char0n.github.io/ramda-adjunct/2.4.1),
+ [2.4.0](https://char0n.github.io/ramda-adjunct/2.4.0),
+ [2.3.0](https://char0n.github.io/ramda-adjunct/2.3.0),
+ [2.2.0](https://char0n.github.io/ramda-adjunct/2.2.0),
+ [2.1.0](https://char0n.github.io/ramda-adjunct/2.1.0),
+ [2.0.0](https://char0n.github.io/ramda-adjunct/2.0.0),
+ [1.19.0](https://char0n.github.io/ramda-adjunct/1.19.0),
+ [1.18.0](https://char0n.github.io/ramda-adjunct/1.18.0),
+ [1.17.0](https://char0n.github.io/ramda-adjunct/1.17.0),
+ [1.16.0](https://char0n.github.io/ramda-adjunct/1.16.0),
+ [1.15.0](https://char0n.github.io/ramda-adjunct/1.15.0),
+ [1.14.0](https://char0n.github.io/ramda-adjunct/1.14.0),
+ [1.13.0](https://char0n.github.io/ramda-adjunct/1.13.0),
+ [1.12.0](https://char0n.github.io/ramda-adjunct/1.12.0),
+ [1.11.0](https://char0n.github.io/ramda-adjunct/1.11.0),
+ [1.10.2](https://char0n.github.io/ramda-adjunct/1.10.2),
+ [1.10.1](https://char0n.github.io/ramda-adjunct/1.10.1),
+ [1.10.0](https://char0n.github.io/ramda-adjunct/1.10.0),
+ [1.9.0](https://char0n.github.io/ramda-adjunct/1.9.0),
+ [1.8.1](https://char0n.github.io/ramda-adjunct/1.8.1),
+ [1.8.0](https://char0n.github.io/ramda-adjunct/1.8.0),
+ [1.7.0](https://char0n.github.io/ramda-adjunct/1.7.0),
+ [1.6.0](https://char0n.github.io/ramda-adjunct/1.6.0),
+ [1.5.0](https://char0n.github.io/ramda-adjunct/1.5.0),
+ [1.4.0](https://char0n.github.io/ramda-adjunct/1.4.0),
+ [1.3.2](https://char0n.github.io/ramda-adjunct/1.3.2),
+ [1.3.1](https://char0n.github.io/ramda-adjunct/1.3.1),
+ [1.3.0](https://char0n.github.io/ramda-adjunct/1.3.0),
+ [1.2.0](https://char0n.github.io/ramda-adjunct/1.2.0),
+ [1.1.0](https://char0n.github.io/ramda-adjunct/1.1.0),
+ [1.0.0](https://char0n.github.io/ramda-adjunct/1.0.0),
+ [0.7.0](https://char0n.github.io/ramda-adjunct/0.7.0),
+ [0.6.0](https://char0n.github.io/ramda-adjunct/0.6.0),
+ [0.5.1](https://char0n.github.io/ramda-adjunct/0.5.1),
+ [0.4.0](https://char0n.github.io/ramda-adjunct/0.4.0),
+ [0.3.0](https://char0n.github.io/ramda-adjunct/0.3.0),
+ [0.2.0](https://char0n.github.io/ramda-adjunct/0.2.0),
+ [0.1.0](https://char0n.github.io/ramda-adjunct/0.1.0),
+ [0.0.1](https://char0n.github.io/ramda-adjunct/0.0.1),
```

Comment:
remove comma

Reply:
Thanks. If there is one thing that constantly surprises me about this line of work, it's how easy it is to miss the glaringly obvious.

Reply:
yeah, in programming, mistakes are ~always~ often dumb. That's why code review is a very good thing to do ;-)